power_grid_model_c/power_grid_model/include/power_grid_model/component/source.hpp

```diff
@@ -28,8 +28,11 @@ class Source : public Appliance {
     }
 
     explicit Source(SourceInput const& source_input, double u)
-        : Appliance{source_input, u}, u_ref_{source_input.u_ref}, u_ref_angle_{}, y1_ref_{}, y0_ref_{} {
-        u_ref_angle_ = is_nan(source_input.u_ref_angle) ? 0.0 : source_input.u_ref_angle;
+        : Appliance{source_input, u},
+          u_ref_{source_input.u_ref},
+          u_ref_angle_{is_nan(source_input.u_ref_angle) ? 0.0 : source_input.u_ref_angle},
+          y1_ref_{},
+          y0_ref_{} {
```

Comment:
to get rid of the sonar cloud issue, you can probably modify the statement where this variable is created and add `{}` before the `;`, like I did in this commit: https://github.com/PowerGridModel/power-grid-model/pull/296/commits/8e5b2954020014399b6294f741c5f681579e3597

Reply:
resolved in favour of https://github.com/PowerGridModel/power-grid-model/pull/302/files#r1260654721